### PR TITLE
Prevented shift cancellation after shift has started

### LINKED ISSUE #321: Don’t allow shift cancellation once a shift has started
Suggested by @smarshy.

### PR DESCRIPTION
**Pull request by FadedCoder**
Fixes #321 .

### CODE REVIEW

**Comment by InfernoCoder** [comment]
I always forget about these tests... Uploading a fix soon.

**Comment by InfernoCoder** [comment]
@smarshy Pushed the changes. Now, this only shows the message if the button is clicked and shift is running.

**Comment by smarshy** [comment]
@InfernoCoder Hi, able to cancel the shift; can you list the steps you used to verify?

**Comment by InfernoCoder** [comment]
@smarshy I have made some changes in the code. But this code will run only if the shift was created in, and the check was done in the same time-zone. The timezone of the person who creates the shifts is not registered, so it is impossible to compare two different time-zones To check this code, please enter your timezone in the `settings.py`, in `TIME_ZONE = '<YOUR TIMEZONE>'`, under `# Internationalization`.

This is the pic of the cancellation confirmation page -
ss

**Comment by InfernoCoder** [comment]
@smarshy I have fixed the code perfectly now. Djang I o used to override the default timezone settings, but I fixed that with the `tzlocal` module. It works perfectly now, and here is the screenshot -
ss

**Comment by tapasweni-pathak** [comment]
@InfernoCoder Can you please resolve the conflicts?

**Comment by InfernoCoder** [comment]
@tapasweni-pathak I deleted my forked branch SHIFT_CANCEL, and so, I can't modify. What should I do?

**Comment by smarshy** [comment]
@InfernoCoder You can do these steps:
1) clone this pull request locally
2) checkout a new branch from it
3) make your changes in that and raise the corresponding PR
See the modifying inactive pull requests section in https://help.github.com/articles/checking-out-pull-requests-locally/

File: vms/shift/views.py [hunk 1]
```diff
@@ -18,6 +18,9 @@
 from django.views.generic import ListView
 from django.utils.decorators import method_decorator
 from django.core.urlresolvers import reverse_lazy
+from tzlocal import get_localzone
```
Comment (tapasweni-pathak) [inline vms/shift/views.py]: @FadedCoder Did you try without tzlocal?

**Pull request closed without merging**